app/views/content/life-as-a-teacher/explore-subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -3,52 +3,80 @@
-     </p>
-
     <p>Your training provider may offer part-time teacher training courses.</p>
 
      <p>You'll also be invited to an annual community day and have the opportunity to network with existing physics teachers.
      </p>
 
     <p><a href="https://find-teacher-training-courses.service.gov.uk/results?engineers_teach_physics=true&study_type%5B%5D=full_time&study_type%5B%5D=part_time&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&degree_required=two_two&applications_open=false&applications_open=true&l=2&subjects%5B%5D=F3">Find engineers teach physics teacher training courses</a>.
      </p>
-
-    <%= render Content::QuoteComponent.new(
-      text: "I've found the skills I learnt in engineering have stood me in good stead for a teaching career. I have never regretted my decision to go into this profession. ",
-      name: "Chris, physics teacher",
-      large: false
-    ) %>
+    <p>You can also <a href="https://find-teacher-training-courses.service.gov.uk/">search for physics teacher training courses</a> that are not part of the Engineers teach physics training programme.</p>
+    <%= render 'content/shared/quotes/quote_engineers_teach_physics_chris' %>
   </section>
 
-  <section class="col col-720">
-    <h2 class="heading--box-blue">What you'll learn</h2>
-    <h3>On this initial teacher training (ITT) course, you’ll:</h3>
+   <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">national curriculum for physics</a>, with opportunities to develop your pupils' ability to think like physicists and see physics as a solution to many challenges.</p>
+    <p>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</p>
+    <ul>
+      <li>energy</li>
+      <li>motion and forces</li>
+      <li>electricity and electromagnetism</li>
+      <li>waves and matter</li>
+    </ul>
+    <p>Themes you’ll cover when you teach 14 to 16 year olds (key stage 4) include:</p>
     <ul>
-      <li>learn more about the curriculum you’ll be teaching</li>
-      <li>develop your existing physics subject knowledge</li>
-      <li>gain valuable classroom experience</li>
+      <li>energy, forces and wave motion</li>
+      <li>electricity, magnetism and electromagnetism</li>
+      <li>the structure of matter</li>
+      <li>atomic structure and space physics</li>
     </ul>
+    <p>Explore what teaching a key stage 4 physics lesson would be like at the <a href="https://www.thenational.academy/teachers/programmes/physics-secondary-ks4-higher-aqa/units">Oak National Academy website</a>.</p>
```

Comment:
As per previous subject pages, we should avoid pointing to just one exam board?

The Physics page also needs updating to point to multiple boards (other subjects might too - can you check and create a new ticket for this that covers them all?)

Reply:
Resolving this conversation as it will be covered under a separate ticket